README.md

```diff
@@ -37,6 +37,8 @@ These don't matter except for other assembly patches
     - hooks/CameraPerf.cpp
 - Optimised some AI actions
     - hooks/aiinitattack.cpp
+- Optimized lua tables filling
```

Comment:
What about: `Reduce overhead of population of Lua tables in C code`

Reply:
And perhaps mention the functions that are impacted too. I'd like to see if I can measure a difference 😄 !

Reply:
Do you need exe to test?

Reply:
No, I'll put it on FAF Develop and test through there.